src/components/entries/FEEL/FeelPopup.js

```diff
@@ -34,16 +36,26 @@ export default function FEELPopupRoot(props) {
   const [ source, setSource ] = useState(null);
   const [ sourceElement, setSourceElement ] = useState(null);
 
-  const handleOpen = (key, config, _sourceElement) => {
-    setSource(key);
+  const emit = (type, context) => {
+    eventBus.fire('feelPopup.' + type, context);
```

Comment:
If we want to keep backwards compatibility then we'd need to make this optional.

```suggestion
    eventBus && eventBus.fire('feelPopup.' + type, context);
```